positron/modules/atom_browser_app.js

```diff
@@ -4,7 +4,20 @@
 
 "use strict";
 
-exports.app = { /* stub */ };
+const Cc = Components.classes;
+const Ci = Components.interfaces;
+const Cr = Components.results;
+const Cu = Components.utils;
+
```

Comment:
> Also, are all those used?

They aren't, but I'd prefer to make this boilerplate consistent across modules that access Components.